requirements/base.txt

```diff
@@ -2,7 +2,6 @@
 elasticsearch==2.1.0
 Django==1.9.8
 djangorestframework==3.4.0
-psycopg2==2.6.2
```

Comment:
dev settings uses sqlite inherited from https://github.com/kinaklub/next.filmfest.by/blob/80b6c23d8c84d78ec7b6b7b263f7ddc84d326a63/filmfest/settings/base.py#L96 so local configuration doesn't use it.